types/pull-stream/sources/count.d.ts

```diff
@@ -3,5 +3,5 @@ import pull = require('..');
 /**
  * Create a stream that outputs 0 ... `max`. By default, `max` = `Infinity`.
  */
-declare function count(max?: number, onAbort?: (err?: Error | null) => unknown): pull.Source<number>;
+declare function count(max?: number): pull.Source<number>;
```

Comment:
These typings are now more accurate documentation than the package docs...
Docs: https://github.com/pull-stream/pull-stream/blob/master/docs/sources/count.md
Source: https://github.com/pull-stream/pull-stream/blob/master/sources/count.js

Nice find! 👍